application/src/main/java/org/opentripplanner/updater/siri/SiriTimetableSnapshotSource.java

```diff
@@ -228,6 +230,16 @@ private Result<TripUpdate, UpdateError> handleModifiedTrip(
       return UpdateError.result(trip != null ? trip.getId() : null, NOT_MONITORED, dataSource);
     }
 
+    for (var call : CallWrapper.of(estimatedVehicleJourney)) {
```

Comment:
This will only be validated for modified trips, but I think we should probably enforce it for added trips as well? What do you think about moving this check to it's own method and calling it as the first thing in the `SiriTimetableSnapshotSource.apply()` method?

Reply:
That's a good point, since there it is even more important. I changed it here: https://github.com/opentripplanner/OpenTripPlanner/pull/6266/commits/5315ec1ca2432d8a2b2e5e3ab86d7d43cdbb2aef

Reply:
I also added a parameterized test so both options are tested.